src/main/webapp/i18n/en/repository.json

```
@@ -20,6 +20,17 @@
                     "empty": "No changes",
                     "fileUnchanged": "No changes in the file content"
                 }
+            },
+            "vcsAccessLog": {
+                "title": "VCS Access Log",
+                "openVcsAccessLog": "Open Access Log",
+                "userId": "User Id",
+                "author": "Author",
+                "actionType": "Action Type",
+                "authMechanism": "Authentication",
```

```suggestion
                "authMechanism": "Accesstype",
```
or `Authenticationtype` but I think just `Authentication` is too weak